fine-tuning.md

```diff
@@ -19,19 +19,63 @@ Some ideas:
 For bespoke applications, models can be trained on task-specific data. However, training a model from scratch is seldom required.
 The model has already learned useful feature representations during its initial (pre) training, so it is often sufficient to simply fine-tune. This takes advantage of [transfer learning](https://www.v7labs.com/blog/transfer-learning-guide), producing better task-specific performance with minimal training examples & resources -- analogous to teaching a university student without first reteaching them how to communicate.
 
-## How Fine-Tuning Works
+## Transfer Learning versus Fine-tuning
 
-1. Start with a pre-trained model that has been trained on a large generic dataset.
-2. Take this pre-trained model and add a new task-specific layer/head on top. For example, adding a classification layer for a sentiment analysis task.
-3. Freeze the weights of the pre-trained layers so they remain fixed during training. This retains all the original knowledge.
-4. Only train the weights of the new task-specific layer you added, leaving the pre-trained weights frozen. This allows the model to adapt specifically for your new task.
-5. Train on your new downstream dataset by passing batches of data through the model architecture and comparing outputs to true labels.
-6. After some epochs of training only the task layer, you can optionally unfreeze some of the pre-trained layers weights to allow further tuning on your dataset.
-7. Continue training the model until the task layer and selected pre-trained layers converge on optimal weights for your dataset.
+Both {term}`transfer learning` and {term}`fine-tuning` modify a pre-trained model for a domain/task-specific use, and thus both terms are often used interchangeably. However, there are key differences.
 
-The key is that most of the original model weights remain fixed during training. Only a small portion of weights are updated to customise the model to new data. This transfers general knowledge while adding task-specific tuning.
+```{table} Transfer Learning versus Fine-tuning
+Description | Transfer Learning | Fine-tuning
+------------|-------------------|------------
+Based on a model pre-trained on a large generic dataset | yes | yes
+Freeze pre-trained model layers | most or all | none ("full" fine-tuning) or a few
+Head layer | append a new head | replace existing head or leave as-is
+Train on domain-specific data until unfrozen layers converge | yes | yes
+```
+
+### Transfer Learning
+
+The key is that "frozen" layers remain unchanged -- retaining the original abilities of the pre-trained model -- and act as general & robust feature extractors.
+
+```{figure-md} transfer-learning-architecture
+:class: caption
+![](https://static.premai.io/book/transfer_learning.png)
```

Comment:
change (consistent capitalisation and hyphens):

- "domain specific input data" -> "domain-specific input data"
- "pretrained model" -> "pre-trained model"
- "Domain specific Model" -> "domain-specific model"

consistency with other diagram:

- pre-trained model: red colour
- domain-specific model: green colour
- add a "prediction" output box